release-scripts/getUpdatedVersions.js

```diff
@@ -0,0 +1,80 @@
+const { spawn } = require("child_process");
+const fs = require('fs');
+const process = require('process')
+
+async function runProcess(command, args) {
+    const p = spawn(command, args)
+    let output = '';
+    p.stdout.on("data", data => {
+        output += data.toString()
+    });
+
+
+    return new Promise((resolve, reject) => {
+        p.on('exit', (code) => {
+            if (code !== 0) {
+                return reject(code);
+            }
+
+            resolve(output)
+        })
+    });
+}
+
+const VERSION_DIFF_REGEX = /^\+\s*"version":\s*"([0-9\.]+)".*$/
```

Comment:
Version might not just be digits. It could also contain `alpha`, `beta`, etc. for pre-release libs.